java/common/src/main/java/org/ray/util/MethodId.java

```diff
@@ -0,0 +1,199 @@
+package org.ray.util;
+
+import com.google.common.base.Preconditions;
+import java.io.Serializable;
+import java.lang.invoke.MethodHandleInfo;
+import java.lang.invoke.SerializedLambda;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.WeakHashMap;
+import org.objectweb.asm.Type;
+import org.ray.util.logger.RayLog;
+
+
+public final class MethodId {
```

Comment:
Can we add a small comment on what this class is doing? I.e. it seems to be similar to our FunctionID in Ray, except it also has a class associated with it? Also, let's document the fields methodDesc and encoding, it's not clear to me at the moment what they are.

Reply:
ok, I would add the doc